sdk/confluent/Microsoft.Azure.Management.Confluent/src/Generated/Models/ConfluentAgreementResource.cs

```diff
@@ -48,11 +50,12 @@ public ConfluentAgreementResource()
         /// <param name="signature">Terms signature.</param>
         /// <param name="accepted">If any version of the terms have been
         /// accepted, otherwise false.</param>
-        public ConfluentAgreementResource(string id = default(string), string name = default(string), string type = default(string), string publisher = default(string), string product = default(string), string plan = default(string), string licenseTextLink = default(string), string privacyPolicyLink = default(string), System.DateTime? retrieveDatetime = default(System.DateTime?), string signature = default(string), bool? accepted = default(bool?))
+        public ConfluentAgreementResource(string id = default(string), string name = default(string), string type = default(string), SystemData systemData = default(SystemData), string publisher = default(string), string product = default(string), string plan = default(string), string licenseTextLink = default(string), string privacyPolicyLink = default(string), System.DateTime? retrieveDatetime = default(System.DateTime?), string signature = default(string), bool? accepted = default(bool?))
```

Comment:
This is a breaking change; new parameters should be at the end. Please rearrange the parameters in the swagger.

Reply:
Hi @JonathanCrd, SystemData is at the end only - https://github.com/Azure/azure-rest-api-specs/blob/f69f2e8053ac8ae352b11c33655234dea4cf4cfd/specification/confluent/resource-manager/Microsoft.Confluent/preview/2021-03-01-preview/confluent.json#L585